ketcher-autotests/tests/utils/canvas/helpers.ts

```diff
@@ -123,6 +196,18 @@ export async function takeTopToolbarScreenshot(page: Page) {
   await expect(editor).toHaveScreenshot();
 }
 
+export async function takeBottomToolbarScreenshot(page: Page) {
+  page.getByTestId('bottom-toolbar');
```

Comment:
This function doesn't do what it supposed to. 
```
page.getByTestId('bottom-toolbar');
```
Literally does nothing

Reply:
Done.